docs/rules/no-named-import.md

```diff
@@ -0,0 +1,60 @@
+# Prevent named imports from react (react/no-named-import)
+
+Prevent the use of named imports from React, so it's visible in code that it's React and not custom code. This rule
+also creates consistency inside a codebase because some people use named imports and some don't.
+
+## Rule Details
+
+### Options
+
+This rule has two different properties that can be used (not at the same time): `whitelist` and `blacklist`. Both are an array of strings. By default this rule whitelists nothing, so every named import will error.
+
+Example configurations:
+
+```javascript
+// Blacklist
+{
+  "rules": {
+    "react/no-named-import": ["error", {
+      "blacklist": ["Component", "useEffect"]
```

Comment:
"whitelist" and "blacklist" are not inclusive terms; please change it to something like "only" vs "except"?

Reply:
I've changed it to `allow` and `forbid`